modules/howtos/pages/provisioning-cluster-resources.adoc

```diff
@@ -74,7 +74,53 @@ The `flush_bucket()` operation may fail if the bucket does not have flush enable
 
 == Collection Management
 
-This is a xref:6.@server:developer-preview:collections/collections-overview.adoc[Developer Preview feature], see the https://docs.couchbase.com/sdk-api/couchbase-ruby-client/Couchbase/Management/CollectionManager.html[API doc].
+[.status]#Developer Preview#
+
+The CollectionManager interface may be used to create and delete scopes and collections from the Couchbase cluster.
+It is instantiated through the `Bucket.collections()` method.
+Refer to the https://docs.couchbase.com/sdk-api/couchbase-ruby-client/Couchbase/Management/CollectionManager.html[`CollectionManager` API documentation]
+for further details.
+
+[source,ruby]
+----
+include::example$provisioning-resources-collections.rb[tag=create-collection-manager,indent=0]
+----
+
+You can create a scope:
+
+[source,java]
```

Comment:
Should be `ruby` here, same for the others below.

Reply:
oops! ta